### ISSUE TITLE
`minimum_resolution` tests failure due to script not considering scaling

### ISSUE THREAD

**Issue by rene-oromtz** [canonical/checkbox#2178]
### Bug Description

I have a couple of desktop/workstation environments that are failing the minimum resolution tests. From the `xrandr` output I can see both attached devices are detected as 1920 x 1080 however, while running the tests, the resolution is halved as its scaled by 2.

Should scaling be considered while comparing the minimum resolution? Or is this a environment issue most likely with the emulator?

### Cert-blocker Test Case

- [ ] cert-blocker

### To Reproduce

Run `com.canonical.certification::sru`

### Expected Result

Minimum resolution should pass

### Actual Result

Resolution is detected by the script as 960 x 540

### Environment

OS: plucky kernel 6.14.0-36-generic
Checkbox Version: 7.0.0~dev26~ubuntu25.04.1

Submissions: 
https://certification.canonical.com/hardware/202008-28111/submission/456321/
https://certification.canonical.com/hardware/202102-28727/submission/456320/

### Relevant log output

```shell
/tmp/nest-bi76qsfg.d0a260601df2f40e8929b34d387eb1b34ec5528812dccb31cc2977d5339eea3a/resolution_test.py:24: DeprecationWarning: Gdk.Screen.get_primary_monitor is deprecated
  geom = screen.get_monitor_geometry(screen.get_primary_monitor())
/tmp/nest-bi76qsfg.d0a260601df2f40e8929b34d387eb1b34ec5528812dccb31cc2977d5339eea3a/resolution_test.py:24: DeprecationWarning: Gdk.Screen.get_monitor_geometry is deprecated
  geom = screen.get_monitor_geometry(screen.get_primary_monitor())
Minimum acceptable display resolution: 800 x 600
Detected display resolution: 960 x 540
```

### Additional context

Available monitors
```
xrandr --listmonitors
Monitors: 2
 0: +*HDMI-1 1920/700x1080/390+1920+0  HDMI-1
 1: +eDP-1 1920/340x1080/190+0+0  eDP-1
```

Manually modify `check_resolution` to also include scaling for verification:
```
check_resolution()
<python-input-4>:5: DeprecationWarning: Gdk.Screen.get_monitor_scale_factor is deprecated
Monitor 1:
  960 x 540, scale = 2
Monitor 2:
  1920 x 1080, scale = 1
```
